notebooks/223-text-prediction/223-text-prediction.ipynb

```diff
@@ -0,0 +1,773 @@
+{
```

Comment:
<p>Header is not clear, maybe "Create PyTorch model" will be better?</p><p>Also please add small explanation like:</p><p>the code bellow demonstrates how to create model class and download pretrained model using huggingface transformer library. </p><p> </p><p>Also transformers has good feature AutoModel and AutoTokeinizer which can simplify your code below, maybe it is better to use it instead of reference specific classes?</p><p> </p><p>it maybe just use one variable model_id with selected pretrained model name and other things use the same code</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/897/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>